Dockerfile

```diff
@@ -3,11 +3,15 @@ FROM python:2.7
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
-# tox isn't defined in requirements.txt because it needs to be installed globally
-# The dependencies in requirements.txt are only used for tests and are
-# installed by tox at runtime
+# tox isn't defined in requirements-test.txt because it needs to be installed
+# globally. The dependencies in requirements-test.txt are only used for tests
+# and are installed by tox at runtime
 RUN pip install --no-cache-dir tox
 
+# pre-install exact dependency versions using pip
+COPY requirements.txt /usr/src/app/
```

Comment:
Do we need to copy in both `requirements.txt` and `requirements-test.txt` here? Otherwise how will the tests run in Docker?